WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -346,4 +345,35 @@ + (NSRange)bindingPortRangeFromArguments
   return NSMakeRange(port, 1);
 }
 
++ (void)setReduceMotionEnabled:(BOOL)isEnabled
+{
+  if (isSDKVersionLessThan(@"10.0")) {
+    // setReduceMotionEnabled exists over iOS 10
```

Comment:
so, should it  be the SDK version comparison or iOS version comparison? In general, I'd would prefer to avoid such checks and rely on target classes/methods presence (aka `AXSettings -> setReduceMotionEnabled`)

Reply:
Yes, true > rely on target classes/methods presence